libkbfs/disk_limits_windows.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libkbfs
+
+import (
+	"unsafe"
+
+	"github.com/pkg/errors"
+	"golang.org/x/sys/windows"
+)
+
+// getDiskLimits gets a diskLimits object for the logical disk
+// containing the given path.
+//
+// TODO: Also return available files.
+func getDiskLimits(path string) (availableBytes uint64, err error) {
+	pathPtr, err := windows.UTF16PtrFromString(path)
+	if err != nil {
+		return 0, errors.WithStack(err)
+	}
+
+	dll := windows.NewLazySystemDLL("kernel32.dll")
```

Comment:
According to the package docs, the method used by go-disk-usage is vulnerable to DLL preloading attacks.